x-pack/legacy/plugins/code/common/repo_file_status.ts

```diff
@@ -84,6 +85,12 @@ export const RepoFileStatusText = {
       defaultMessage: 'Current file is covered by dedicated language server.',
     }
   ),
+  [RepoFileStatus.LANG_SERVER_LAUNCH_FAILED]: i18n.translate(
+    'xpack.code.repoFileStatus.langServerLaunchFailed',
```

Comment:
is this a new i18n item? should we add a zh-cn item?

Reply:
Yes, where is the zh-cn item?

Reply:
this one: https://raw.githubusercontent.com/elastic/kibana/master/x-pack/plugins/translations/translations/zh-CN.json